packages/components/src/Lettera.purs

```diff
@@ -218,7 +269,7 @@ useResponse f (Right response)
                                     }
       Left err -> LetteraResponse { maxAge: Nothing, body: Left err }
   | otherwise =
-      pure $ LetteraResponse { maxAge: Nothing, body: Left $ HttpError $ unwrap $ response.status }
+      pure $ LetteraResponse { maxAge: Nothing, body: Left $ HttpError request $ stringify (encodeResponse response) }
```

Comment:
`stringify` and `encodeResponse` are only needed because `Json` was used as the type. Remove them.

Reply:
I thought using JSON would be consistent with our previous decision to add JSON logging for incoming requests.